### ISSUE TITLE
Issue with .Net Core version 3.0

### ISSUE THREAD

**Issue by FrantzUml** [dotnet/BenchmarkDotNet#1285]
With BenchmarkDotNet version V0.11.5, and a solution using Core 3.0, the benchmark cannot find the dll.
This one is in the same folder as the executable.
BenchMark searchs on the C: drive as the executable is on the D:

**Comment by adamsitnik** [comment]
Hi @FrantzUml 

We need a full log to be able to help you.

**Comment by Tomcooper12** [comment]
Taking some guessing as to what this issue is but I had a similar experience getting the following error on a fresh .NET core 3 app:

```
// Validating benchmarks:                                                                                                                                          
// ***** BenchmarkRunner: Start   *****                                                                                                                            
// ***** Found 2 benchmark(s) in total *****                                                                                                                       
// ***** Building 1 exe(s) in Parallel: Start   *****                                                                                                              
// start dotnet restore  /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 in C:\source\repos\ConsoleApp3\ConsoleApp3\bin\Release\netcoreapp3
.0\326fc3d5-b31d-43c9-9e27-787b0de281d1                                                                                                                            
// command took 3.23s and exited with 1                                                                                                                            
// ***** Done, took 00:00:03 (3.61 sec)   *****                                                                                                                    
// Found 2 benchmarks:                                                                                                                                             
//   IntroBasic.Sleep: DefaultJob                                                                                                                                  
//   IntroBasic.Thread.Sleep(10): DefaultJob                                                                                                                       
Setup power plan (GUID: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c FriendlyName: High performance)// Build Error: Standard output:                                       
                                                                                                                                                                  
 Standard error:                                                                                                                                                   
   Restore failed in 282.52 ms for C:\source\repos\ConsoleApp3\ConsoleApp3\bin\Release\netcoreapp3.0\326fc3d5-b31d-43c9-9e27-787b0de281d1\BenchmarkDotN
et.Autogenerated.csproj.                                                                                                                                           
C:\source\repos\ConsoleApp3\ConsoleApp3\bin\Release\netcoreapp3.0\326fc3d5-b31d-43c9-9e27-787b0de281d1\BenchmarkDotNet.Autogenerated.csproj : error NU1
201: Project ConsoleApp3 is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1). Project ConsoleApp3 supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)   
  Restore completed in 70.56 ms for C:\source\repos\ConsoleApp3\ConsoleApp3\ConsoleApp3.csproj.                                                        
                                                                                                                                                                   
// Build Error: Standard output:                                                                                                                                   
                                                                                                                                                                   
 Standard error:                                                                                                                                                   
   Restore failed in 282.52 ms for C:\source\repos\ConsoleApp3\ConsoleApp3\bin\Release\netcoreapp3.0\326fc3d5-b31d-43c9-9e27-787b0de281d1\BenchmarkDotN
et.Autogenerated.csproj.                                                                                                                                           
C:\source\repos\ConsoleApp3\ConsoleApp3\bin\Release\netcoreapp3.0\326fc3d5-b31d-43c9-9e27-787b0de281d1\BenchmarkDotNet.Autogenerated.csproj : error NU1
201: Project ConsoleApp3 is not compatible with netcoreapp2.1 (.NETCoreApp,Version=v2.1). Project ConsoleApp3 supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)   
  Restore completed in 70.56 ms for C:\source\repos\ConsoleApp3\ConsoleApp3\ConsoleApp3.csproj.       
```

Updating the targets in the csproj allows the core 3 app to run but it will still benchmark against core 2.1 by the looks of things:

```
<TargetFrameworks>netcoreapp3.0;netcoreapp2.1</TargetFrameworks>
```

**Comment by adamsitnik** [comment]
https://github.com/dotnet/BenchmarkDotNet/pull/1233 improved the .NET Core version detection

We have published a `0.12.0` version with the fix I 1h ago. Please update.
